HTTPServer.driver/BadRequestFilter.cs

```diff
@@ -22,7 +24,10 @@ public Reply Execute(Request request)
                 reply.StartingLine = Encoding.UTF8.GetBytes("HTTP/1.1 400 Bad Request\r\n");
                 return reply;
             }
-            return _nextCommand.Execute(request);
+            var returnedReply = _nextCommand.Execute(request);
+            Logger logger = new Logger();
+            logger.LogData(logger.LogMessage(request, returnedReply));
```

Comment:
I wasn't thrilled with putting it here, but it seemed like the best of all of the options.